index.html

```diff
@@ -395,11 +415,11 @@
 				<p>The availability of this address does not preclude the creation and use of other identifiers and/or
 					addresses to retrieve a representation of a <a>Web Publication</a> in whole or part.</p>
 
-				<div class="note">The Web Publication's <a href="#pub-address">address</a> can also be used as value for an
-					identifier link relation [[link-relation]].</div>
+				<div class="note">The Web Publication's address can also be used as value for an identifier link relation
```

Comment:
If we have a canonical identifier, it probably makes more sense to use the canonical identifier instead of the address in `link@rel="identifier"`.

The `identifier` value is just a proposal at this point, not sure that we should talk about it at all.

Reply:
Could you open as a new issue? I'm working on consolidating the three PRs and reverting the defs so will be closing these off shortly.